math/scalapack/Portfile

```diff
@@ -8,8 +8,9 @@ PortGroup           github 1.0
 PortGroup           mpi 1.0
 PortGroup           muniversal 1.0
 
-github.setup        Reference-ScaLAPACK scalapack 2.2.2 v
-revision            0
+version             2.2.2
+github.setup        Reference-ScaLAPACK scalapack ${version} v
```

Comment:
no need to define "version" separately, it should just remain part of the "github.setup" line

Reply:
ah. I wasn't aware of its creation within github.setup. Kind of neat.